pyuploadcare/api_resources.py

```diff
@@ -231,12 +231,28 @@ def store(self):
         """
         self._info_cache = rest_request('PUT', self._api_storage_uri)
 
-    def copy(self, effects=None, target=None):
+    def copy(self, effects=None, target=None, make_public=None, store=None, pattern=None):
         """Creates File copy
```

Comment:
this will break backwards compatibility (as I have learned the hard way). moving `store` to the fourth place from fifth is reasonable though. `make_public` can be the last as I think that `pattern` will be used more.